### ISSUE TITLE
Visual Bug - Combat log

### ISSUE THREAD

**Issue by silverjade** [HellscreamWoW/Tracker#2501]
Combat log is saying I'm doing far more damage than I actually am doing. I'm doing more like 3k (crit) or less with Melee.

![](http://i.imgur.com/OTDC8Is.jpg)

**Comment by silverjade** [comment]
Also as a note, for some weird reason after I went AFK for several minutes and it dc'd me, I came back in and had to go afk again for a shorter time, and I noticed after I returned my dmg was low until I relogged. Not sure why that happened. 

Ex: My Thrash after returning from afk mode was only doing 2-digit  dmg, after relog it began doing 3-digit dmg again.

**Comment by Guybrush-Hellscream** [comment]
Yeah i noticed this bug a while ago, during the caster damage fix.
http://www.hellscream-wow.com/forum/viewtopic.php?f=8&t=2096
